OracleDatabase/SingleInstance/FAQ.md

```diff
@@ -74,3 +74,7 @@ echo "DISABLE_OOB=ON" >> ~/.sqlnet.ora
 ```
 
 For more information configuring _sqlnet.ora_ file see [Database Net Services Reference](https://www.oracle.com/pls/topic/lookup?ctx=dblatest&id=GUID-2041545B-58D4-48DC-986F-DCC9D0DEC642), [Instant Client Installation for Linux](https://www.oracle.com/database/technologies/instant-client/linux-x86-64-downloads.html), [What is DISABLE_OOB (Out Of Band Break)? (Doc ID 373475.1)](https://support.oracle.com/epmos/faces/DocumentDisplay?id=373475.1) and issue #1352
+
+## ORA-01157: cannot identify/lock data file
+
+This error occurs when the database cannot find a data file (used for tablespaces) that was previously present. This is most likely because the data file has been located outside the volume in a previous container and was hence not persisted. Ensure that when you add tablespaces and/or data files that they are located within the volume location, i.e. $ORACLE_BASE/oradata/$ORACLE_SID, (e.g. `/opt/oracle/oradata/XE`).
```

Comment:
Can you provide an actual sample SQL command that should be used here?

Reply:
No, I cannot. There are so many parameters with the `CREATE TABLESPACE` clause and it depends on so many factors of what the user wants to do, that I'm hesitant to just pick one that might work that users than just copy/paste. It starts with whether the user wants to create a data, temp, or undo tablespace, whether it's a bigfile tablespace or smallfile one, what the tablespace and file(s) name(s) should be, the size, the autoextend parameters, whether the tablespace should be encrypted, compressed, and about another 100 different parameters. Furthermore, a user could happily just put the data files in a separate volume as well, they don't have to go into that particular directory or any other. Users should be aware that if they create files on the file system in a Docker container, these files have to either remain in a volume or are inherently transient by nature. Given that in 5 years of existence of these scripts this has come up only once and only now, I think it is fair to say that most users know that if they type the `CREATE TABLESPACE` clause, they can and probably should lookup that clause in the documentation of the product, rather than copy/pasting it from a FAQ markdown file for Docker scripts.